### PR TITLE
chore: docs: added more docs

### PR DESCRIPTION
**Pull request by dbarrosop**
https://docs-git-docs-postgres-nhost.vercel.app/database/settings
https://docs-git-docs-postgres-nhost.vercel.app/database/extensions
https://docs-git-docs-postgres-nhost.vercel.app/graphql/settings
https://docs-git-docs-postgres-nhost.vercel.app/database/performance

### CODE REVIEW

File: docs/docs/database/plugins.mdx [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+```
+SET ROLE postgres;
+DROP EXTENSION pg_cron;
+```
+
+### Resources
+
+* [GitHub](https://github.com/citusdata/pg_cron)
+
+## hypog
```
Comment (dbarrosop) [inline docs/docs/database/plugins.mdx:84]: typo

File: docs/docs/database/plugins.mdx [hunk 2]
```diff
@@ -0,0 +1,135 @@
+---
+title: 'Plugins'
```
Comment (nunopato) [inline docs/docs/database/plugins.mdx:2]: is there a reason for not calling them by their name, extensions?

File: docs/docs/database/settings.mdx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+Below you can find the official schema (cue) and an example to configure your postgres database:
+
+<Tabs groupId="package-manager">
+  <TabItem value="schema" label="schema" default>
+
+```cue
```
Comment (nunopato) [inline docs/docs/database/settings.mdx:14]: our users and cue... I would either omit cue altogether or show TOML by default (to the left)

**Review by nunopato — APPROVED** [review]
a couple of comments, but lgtm

File: docs/docs/database/plugins.mdx [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+```
+SET ROLE postgres;
+DROP EXTENSION timescaledb;
+```
+
+### Resources
+
+* [GitHub](https://github.com/timescale/timescaledb)
+* [Documentation](https://docs.timescale.com)
+* [Website](https://www.timescale.com)
```
Comment (dbarrosop) [inline docs/docs/database/plugins.mdx:135]: forgot pg_stat_statements

File: docs/docs/database/plugins.mdx (hunk 2 above)
Comment (dbarrosop) [reply docs/docs/database/plugins.mdx:2]: will change

File: docs/docs/database/settings.mdx (hunk 3 above)
Comment (dbarrosop) [reply docs/docs/database/settings.mdx:14]: I will change the default tab for now, we can decide later on. Ideally we'd render some nice docs from the schema but haven't had the time to research how to do that yet.

**Review by onehassan — APPROVED** [review]
LGTM

**Pull request merged**